### PR TITLE
ez - Created ReviewTable Component + Tests

### LINKED ISSUE #20: Frontend - Make ReviewTable Component
Create a ReviewTable component that shows all reviews in the format returned by the backend endpoint described in #5 for retreiving reviews by user.  It should have an optional columns for "Approve" and "Reject" that appear only if a flag "moderatorOptions" is passed as true in the props to the component.  it should also have a Delete column available only if deleteColumn=true is passed.

Part of #2

### PR DESCRIPTION
**Pull request by emiz39**
Closes #20 

Storybook link: https://6736bb860e8adeca15f06b17-bzzeislyxa.chromatic.com/?path=/docs/components-review-reviewtable--docs

If moderatorOption = true, it will show the Approve/Reject Column (shown in the table titled: Three Items Admin With Moderator Options)

Test with storybook since there are a lot of fields which don't all appear in the screenshot
image

### CODE REVIEW

File: frontend/src/main/components/Review/ReviewTable.js [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+  const approveCallback = (cell) => {
+    console.log(`Approved review with id: ${cell.row.values.id}`);
+  };
+
+  const rejectCallback = (cell) => {
+    console.log(`Rejected review with id: ${cell.row.values.id}`);
+  };
+
+  const columns = [
```
Comment (st4lemon) [inline frontend/src/main/components/Review/ReviewTable.js:46]: adjust columns to match fields in Review.java

**Review by st4lemon — CHANGES_REQUESTED** [review]

File: frontend/src/main/components/Review/ReviewTable.js (hunk 1 above)
Comment (emiz39) [reply frontend/src/main/components/Review/ReviewTable.js:46]: fixed

**Review by st4lemon — DISMISSED** [review]
LGTM

**Review by clevermonkey16 — DISMISSED** [review]
LGTM

**Review by pconrad — CHANGES_REQUESTED** [review]
https://ucsb-cs156.github.io/topics/code_reviews/commented_out_code.html#why-should-commented-out-code-be-removed

**Review by clevermonkey16 — APPROVED** [review]
LGTM

**Review by pconrad — APPROVED** [review]
LGTM

**Pull request merged**